packages/expo-document-picker/README.md

```diff
@@ -0,0 +1,83 @@
+# expo-document-picker
```

Comment:
I'm not sure what the consensus is atm, but weren't we supposed to leave out readmes? 🤔

Reply:
Maybe just link to the docs on docs.expo.io? Like I did here: https://github.com/expo/expo/blob/master/packages/expo-task-manager/README.md

Reply:
maybe the best way to do this for now is to link to https://github.com/expo/expo/blob/master/docs/versions/unversioned/sdk/document-picker.md, so it can always point to the latest version of the docs and be an easy place for people to know where to submit a PR

Reply:
I'll go with @tsapeta suggestion in this and update each universal module once a consensus is reached [here](https://github.com/expo/expo/issues/3315).